Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/NativeLibrary.java

```diff
@@ -353,12 +360,13 @@ public static native String GetConfig(String configFile, String Section, String
   /**
    * Begins emulation.
    */
-  public static native void Run(String path, boolean firstOpen);
+  public static native void Run(String path, String gameId, boolean firstOpen);
```

Comment:
`gameId` can get from `SConfig` in C++, so this parameter is unnecessary.

Reply:
Didn't realize that, moved the buttonmanager init down to after the game boots so that the gameid can get pulled.